src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -499,6 +499,17 @@ protected override async Task<bool> AcceptCommand(MessageId id, string method, J
                     }
 
                 // Protocol extensions
+                case "DotnetDebugger.setNextIP":
+                    {
+                        if (!SourceId.TryParse(args["location"]?["scriptId"]?.Value<string>(), out SourceId sourceId))
+                            return false;
+                        var ret = await OnSetNextIP(id, sourceId, args["location"]["lineNumber"].Value<int>(), args["location"]["columnNumber"].Value<int>(), token);
```

Comment:
Can we use `SourceLocation.Parse(args["location"])` here? It will check for missing values too.